src/index.html

```diff
@@ -15,8 +15,104 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
   </head>
-  <body>
-    <h1>Moyo header</h1>
+
+  <body class="first">
```

Comment:
The `class` attribute of the `body` tag does not represent the meaning of the content. Class names should represent the meaning of the content, not the styles or tag names. Please, change it to a more meaningful name.